app/2.0/toolbox/localize.md

```diff
@@ -4,7 +4,7 @@ title: Localization
 
 <!-- toc -->
 
-**The localization behavior is prerelease.** APIs may be subject to change.
+**The localization behavior is prerelease.** APIs may be subject to change. <--- Still prerelease???
```

Comment:
Nope https://github.com/PolymerElements/app-localize-behavior/releases

Fixed in new PR